kitsune/kbadge/models.py

```diff
@@ -124,20 +65,7 @@ def slugify(txt):
     return txt
 
 
-def get_permissions_for(self, user):
-    """Mixin method to collect permissions for a model instance"""
-    pre = 'allows_'
-    pre_len = len(pre)
-    methods = (m for m in dir(self) if m.startswith(pre))
-    perms = dict(
-        (m[pre_len:], getattr(self, m)(user))
-        for m in methods
-    )
-    return perms
-
-
-# We need to do this in django 1.7 because migrations serialize the fields.
-# See https://code.djangoproject.com/ticket/22999
+# This is only retained to ensure migrations don't fail
 @deconstructible
 class UploadTo(object):
```

Comment:
@pmac Curious to see what your thoughts are here. This class is no longer used, but migrations fail if it's not present. Do we only need to keep this around for _this_ migration, or, because it's referenced in the initial migration, do we need to do something more here (e.g. manually edit the initial migration)?

Reply:
I think it's fine to leave it. We have a task to clean up migrations and we can delete it along with that work. Maybe make a note in the migration itself that we should delete that class?